tests/tests/fee_tests.cpp

```diff
@@ -299,11 +299,11 @@ BOOST_AUTO_TEST_CASE(asset_claim_pool_test)
         GRAPHENE_REQUIRE_THROW( claim_pool( alice_id, alicecoin.id, _core(1), core_asset), fc::exception );
 
         // deposit 300 BTS to the fee pool of ALICECOIN asset
-        fund_fee_pool( alice, alicecoin, _core(300).amount );
+        fund_fee_pool( alice_id(db), alicecoin, _core(300).amount );
 
         // Test amount of CORE in fee pools
-        BOOST_CHECK( alicecoin.dynamic_asset_data_id(db).fee_pool == _core(300).amount );
-        BOOST_CHECK( aliceusd.dynamic_asset_data_id(db).fee_pool == _core(100).amount );
+        BOOST_CHECK( alicecoin.dynamic_asset_data_id(db).fee_pool.value == _core(300).amount );
```

Comment:
No. I mean using `BOOST_CHECK_EQUAL` is better than `BOOST_CHECK( a == b )` when possible. Actually my previous comment is not 100% correct because I overlooked `core_prec`.

Reply:
`BOOST_CHECK_EQUAL( alicecoin.dynamic_asset_data_id(db).fee_pool.value, 300*core_prec)` would work.

Reply:
This "would work" change looks ugly. I don't know why it times precision in the first place which makes the tests a bit complicated. Not a big deal anyway, so I think it's fine to revert this change. Thanks.